plugins/resource_management/app/controllers/resource_management/project_resources_controller.rb

```diff
@@ -42,6 +42,8 @@ def show_area(area = nil)
     end
 
     def confirm_reduce_quota
+      puts "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+      pp params
```

Comment:
I believe this can go. :)

Reply:
ups! thanks

and yes it is a requirement from @reimannf to reduce a quota to the usage with less steps